examples/pymc3_howto/sampling_compound_step.ipynb

```diff
@@ -12,10 +12,24 @@
     "# Compound Steps in Sampling\n",
```

Comment:
<p>az.style.use should go in the 2nd cell, somethimes matplotlib doesn't load the style correctly if it's imported in the same cell the style is set (matplotlib is imported by arviz)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/141/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>